assignments/function-conversion.js

```diff
@@ -5,26 +5,51 @@
 // };
 // myFunction();
 
-// let anotherFunction = function (param) {
+//  ANSWER **********************************************
+
+// let myFunction = () => console.log("Function was Invoked");
+// myFunction();
+
+// let anotherFunction = function(param) {
 //   return param;
 // };
 // anotherFunction("Example");
 
-// let add = function (param1, param2) {
+//  ANSWER *******************************************
+
+// let anotherFunction = param => param;
+// console.log(anotherFunction("Example"));
+
+// let add = function(param1, param2) {
 //   return param1 + param2;
 // };
-// add(1,2);
+// add(1, 2);
+
+// ANSWER **************************************
+
+// let add = (param1, param2) => param1 + param2;
+// console.log(add(1, 2));
 
 // let subtract = function (param1, param2) {
 //   return param1 - param2;
 // };
 // subtract(1,2);
 
+// ANSWER ****************************************
+
+// let subtract = (param1, param2) => param1 - param2;
+// console.log(subtract(1, 2));
 
 // Stretch
 
 // exampleArray = [1,2,3,4];
 // const triple = exampleArray.map(function (num) {
 //   return num * 3;
 // });
-// console.log(triple);
\ No newline at end of file
+// console.log(triple);
+
+// ANSWER *******************************************
+
+// exampleArray = [1, 2, 3, 4];
+// const triple = exampleArray.map(num => num * 3);
+// console.log(triple);
```

Comment:
Nice job tackling the stretch question and figuring out single line returns in arrow functions.